website/blog/2023-07-03-ferretdb-v-1-5-0-beta-support-sqlite.md

```diff
@@ -0,0 +1,50 @@
+---
+slug: ferretdb-v-1-5-0-beta-support-sqlite
+title: FerretDB v1.5.0 New Release with Beta-level Support for SQLite
+authors: [alex]
+description: >
+  We are delighted to announce the release of FerretDB v1.5.0 which includes beta-level support for SQLite backend.
+image: /img/blog/ferretdb-v1.5.0.jpg
+tags: [release]
+---
+
+![FerretDB v.1.5.0 includes beta-level support for SQLite](/img/blog/ferretdb-v1.5.0.jpg)
+
+After a great deal of work from the [FerretDB](https://www.ferretdb.io/) team and our open-source community, we are delighted to announce the release of FerretDB v1.5.0, which includes beta-level support for [SQLite](https://www.sqlite.org/) backend.
+
+<!--truncate-->
+
+For [Tigris](https://www.tigrisdata.com/) backend users, please note that this will be the last release with FerretDB support for the Tigris backend.
```

Comment:
We should replace this paragraph with this text: 

"Important note to Tigris users: FerretDB 1.5.0 is the last release of FerretDB which includes support for the Tigris backend.  If you wish to use Tigris, please do not update FerretDB beyond v1.5.0. Earlier versions of FerretDB with Tigris support will still be available on GitHub."